src/components/QueryDrawer.tsx

```diff
@@ -75,6 +77,13 @@ const SavedQuery = ({
         margin: 4,
       }}
     >
+      <Export
+        isOpen={isExportOpen}
+        onClose={() => setIsExportOpen(false)}
+        results={results.map(({ id, ...a }) => a)}
+        parentUid={uid}
+        clearOnClick={clearOnClick}
+      />
```

Comment:
This seems inefficient.  How can I get the state with line 134 to open the `Export` in `Results View`